### ISSUE TITLE
Data for Image2Label experiment

### ISSUE THREAD

**Issue by wangkaihong** [aayushbansal/Recycle-GAN#17]
Hi Aayush,

I am trying to replicate the Image2label experiment in your paper and I found the dataset you release [here](https://www.dropbox.com/s/qhu29y5cx4lyfun/Viper_data.tar.gz?dl=0) contains 56 clips in the training set:
```
>>> sorted(vid)
['001', '002', '003', '004', '005', '006', '007', '008', '009', '010', '011', '012', '013', '014', '015', '016', '017', '018', '019', '020', '021', '022', '023', '024', '030', '031', '032', '033', '034', '035', '036', '037', '038', '039', '044', '045', '046', '047', '048', '049', '050', '051', '052', '053', '054', '055', '056', '057', '058', '059', '060', '061', '062', '070', '071', '072']
>>> len(vid)
56

```
and 21 clips in the test set:
```
>>> vid
{'028', '068', '066', '076', '040', '073', '077', '025', '042', '063', '029', '074', '069', '065', '064', '027', '041', '067', '026', '043', '075'}
>>> vid = list(vid)
>>> len(vid)
21

```
But I think you mentioned in the paper that the data for the Image2Label experiment from Viper consists of 57 videos as training data and 20 as test. Did I go for the wrong dataset or did I miss anything here?

Also, it would be even better if you can release code for the evaluation part. Much thanks in advance!
